### DISCUSSION TITLE
using .background(AnyView(Color.white)) results in an error

### DISCUSSION THREAD

**Discussion by AleenWael** [LucasMucGH/BottomSheet#74] [Q&A] [answered]
I tried adding a background color to the bottom sheet but the compiler throws this error "Cannot convert value of type 'AnyView' to expected argument type '() -> AnyView' "

<img width="662" alt="Screen Shot 2022-03-16 at 6 42 27 PM" src="https://user-images.githubusercontent.com/67905274/158629959-04106d0d-8429-4f1c-9bb9-0a5f16f9c58b.png">

**Comment by lucaszischka** [comment, marked as answer]
Yes, the syntax was changed to a closure in main. Please use
```swift
.background({
    AnyView(
        Color.white
    )
})
```
